cmd/cue-gen/genoapi.go

```diff
@@ -426,7 +426,7 @@ func (x *builder) genOpenAPI(name string, inst cue.Value) (cue.Value, error) {
 			if doc.Text() == "" {
 				continue
 			}
-			if strings.HasPrefix(doc.Text(), "$hide_from_docs") {
+			if strings.Contains(doc.Text(), "$hide_from_docs") {
```

Comment:
what if this line is commented? Is that possible?

Reply:
Not really, the line is already a (protobuf) comment like `// $hide_from_docs`. So there isn't really a mechanism to have a commented-comment

Reply:
ah ok :)